youtube_dl/extractor/lcp.py

```diff
@@ -0,0 +1,149 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none,
+    parse_iso8601
+)
+
+
+class LcpIE(InfoExtractor):
+    IE_NAME = 'LCP'
+    _VALID_URL = r'https?://(?:www\.)?lcp\.fr/(?:[^\/]+/)*(?P<id>[^/]+)'
+
+    _TESTS = [{
+        'url': 'http://www.lcp.fr/la-politique-en-video/schwartzenberg-prg-preconise-francois-hollande-de-participer-une-primaire',
+        'md5': 'aecf5a330cfc1061445a9af5b2df392d',
+        'info_dict': {
+            'id': 'd56d03e9',
+            'url': 're:http://httpod.scdn.arkena.com/11970/d56d03e9_[0-9]+.mp4',
+            'ext': 'mp4',
+            'title': 'Schwartzenberg (PRG) préconise à François Hollande de participer à une primaire à gauche',
+            'upload_date': '20160226',
+            'description': 'Le président du groupe parlementaire radical, républicain, démocrate et progressiste (RRDP) y voit une bonne occasion pour le président de la République de se "relégitimer".',
+            'timestamp': 1456488895
+        }
+    }, {
+        'url': 'http://www.lcp.fr/emissions/politique-matin/271085-politique-matin',
+        'md5': '6cea4f7d13810464ef8485a924fc3333',
+        'info_dict': {
+            'id': '327336',
+            'url': 're:http://httpod.scdn.arkena.com/11970/327336_[0-9]+.mp4',
+            'ext': 'mp4',
+            'title': 'Politique Matin - Politique matin',
+            'upload_date': '20160225',
+            'description': 'Politique Matin - Politique matin',
+            'timestamp': 1456391602
+        }
+    }, {
+        'url': 'http://www.lcp.fr/le-direct',
+        'info_dict': {
+            'title': 'Le direct | LCP Assembl\xe9e nationale',
+            'id': 'le-direct',
+        },
+        'playlist_mincount': 1
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+
+        # Extract the required info of the media files gathered in a dictionary
+        media_dict = self.__extract_from_webpage(display_id, webpage)
+        # Some web pages embed videos from other platforms like dailymotion, therefore we pass on these URLs
+        if not media_dict:
+            return self.url_result(url, 'Generic')
+
+        # All videos are part of a playlist, a single video is also put in a playlist
+        playlist_files_info = media_dict.get('Playlist')
+        if not playlist_files_info:
+            return self.url_result(url, 'Generic')
+
+        media_files_info = playlist_files_info[0]
+        video_formats = self.__get_video_formats(media_files_info)
+        video_thumbnails = self.__get_thumbnails(media_files_info)
+        video_timestamp = parse_iso8601(media_files_info.get('MediaInfo', {}).get('PublishDate'))
+
+        title = self._og_search_title(webpage)
+        description = self._html_search_regex(self._meta_regex('description'), webpage, 'description', group='content', default=title)
```

Comment:
The default for `default` when calling `_html_search_meta` seems to be `NO_DEFAULT` though instead of `None`. I encountered this when testing for pages where the description is missing, so for this page the test also fails when using `_html_search_meta`, that is why I used the `_html_search_regex` here.
I agree with you that it would be better to use `_html_search_meta` here, but it causes problems when the description is missing.

Do you know any workarounds for cases in which the `description` is missing?